quickstarts/kubernetes/coredns/config.yml

```
@@ -0,0 +1,101 @@
+  
+  For instructions on modifying your cluster's CoreDNS configuration see https://coredns.io/2018/01/29/deploying-kubernetes-with-coredns-using-kubeadm/  
+  
+  
+  ### How do I configure my integration to scrape CoreDNS severs?  
+  
+  The approach to getting your Prometheus metrics into New Relic differs depending on which integration you use.  
+  
+  #### Prometheus Remote Write  
+  
+  In addition to setting up your `remote_write` configuration as described [here](https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/install-configure-remote-write/set-your-prometheus-remote-write-integration/), you will need to add the following scrape configuration to your `prometheus.yml` config file:  
+  
+  ```  
+  - job_name: 'kube-dns'  
+    honor_labels: true  
+    kubernetes_sd_configs:  
+      - role: pod  
+    relabel_configs:  
+      - action: keep  
+        source_labels:  
+          - __meta_kubernetes_namespace  
+          - __meta_kubernetes_pod_name  
+        separator: '/'  
+        regex: 'kube-system/coredns.+'  
+      - source_labels:  
+          - __meta_kubernetes_pod_container_port_name  
+        action: keep  
+        regex: metrics  
+      - source_labels:  
+          - __meta_kubernetes_pod_name  
+        action: replace  
+        target_label: instance  
+      - action: labelmap  
+        regex: __meta_kubernetes_pod_label_(.+)  
+  ```  
+  (This helpful Prometheus job defnition sourced from [sysdig blog](https://sysdig.com/blog/how-to-monitor-coredns/))  
+  
+  Then, after reloading Prometheus config, you can check that your CoreDNS pods are appearing under your [Prometheus targets](https://prometheus.io/docs/prometheus/latest/querying/api/#targets).  
+  
+  #### Prometheus OpenMetrics integration  
+  
+  # This section needs some filling out  
+
+summary: |
+  Visualize CoreDNS performance and alert on potential errors. CoreDNS is a critical Kubernetes cluster component and can be difficult to troubleshoot in an error scenario. Download New Relic CoreDNS quickstart to increase cluster visibility.    
+
+level: New Relic
+
+authors:
+  - New Relic
+
+keywords:
+  - kubernetes
+  - dns
+  - k8s
+
+  # Reference to install plans located under /install directory
+  # Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+  # TODO add install plan
+  # installPlans:
+  # - infra-agent-standard
+# Documentation references
+documentation:
+  - name: Installation docs
+    url: docs.newrelic.com
+    description: Description about this doc reference
+  - name: Kubernetes & CoreDNS
+    url: https://kubernetes.io/docs/tasks/administer-cluster/coredns/
+    description: Details on Kubernetes and CoreDNS
+  - name: CoreDNS Prometheus monitoring
+    url: https://coredns.io/plugins/metrics/
+    description: Metric details exposed by Prometheus CoreDNS extension
+
+# Content / Design
+logo: logo.png
```

Hey @tclayton-newr! The check is just failing because this is still referencing a png, updating that should get everything passing.
```suggestion
logo: logo.svg
```